### PR TITLE
Support us

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

**Comment by biodranik** [comment]
https://dea46381.organicmaps.pages.dev/support-us/

File: content/support-us/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  preview_image: support-us/support-us.jpg
+---
+
+Organic Maps is a free, open-source app. It is free from ads, it does not collect your personal information,
+and it is developed by enthusiasts with the help of the community.
+
+There are different ways to support the development:
+
+- [Donate](@/donate/index.md)! Every dollar or euro counts and helps us to pay for servers and scale.
+- Report bugs and share ideas on our [Github](https://github.com/organicmaps/organicmaps/issues)
```
Comment (rtsisyk) [inline content/support-us/index.md:14]: I don't see code contributions.

**Review by vng — APPROVED** [review]

File: content/support-us/index.md (hunk 1 above)
Comment (biodranik) [reply content/support-us/index.md:14]: See the next line below.

**Comment by biodranik** [comment]
PTAL in a few minutes

**Pull request merged**